Zend/zend_language_scanner.l

```diff
@@ -2346,28 +2519,52 @@ heredoc_scan_done:
 				}
 				/* fall through */
 			case '\n':
+				indentation = spacing = 0;
+
+				while (YYCURSOR < YYLIMIT && (*YYCURSOR == ' ' || *YYCURSOR == '\t')) {
+					if (*YYCURSOR == '\t') {
+						spacing |= HEREDOC_USING_TABS;
+					} else {
+						spacing |= HEREDOC_USING_SPACES;
+					}
+					++YYCURSOR;
+					++indentation;
+				}
+
+				if (YYCURSOR == YYLIMIT) {
+					CG(increment_lineno) = 1;
```

Comment:
This case does not initialize `zendlval`.

Reply:
Done (as well as in a couple of other places), thanks!